consumers/src/main/java/uk/gov/moj/sdt/consumers/util/ClaimDefencesResultsUtil.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.moj.sdt.consumers.util;
+
+import org.springframework.stereotype.Component;
+import uk.gov.moj.sdt.cmc.consumers.model.ResponseType;
+import uk.gov.moj.sdt.cmc.consumers.model.claimdefences.ClaimDefencesResult;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class ClaimDefencesResultsUtil {
+
+    public List<ClaimDefencesResult> createClaimDefencesList() {
```

Comment:
This method looks like it creates dummy data - is it needed?

Reply:
it is used in test cases

Reply:
Would be better to separate out test code from production code.

This particular method is used in some consumer unit tests and the MockClaimDefencesService, the latter being part of the production code.  I think it would be better to have a private method in MockClaimDefencesService to create the claim defence results, then this code can be moved to a new ClaimDefencesResultsTestUtil class in the consumers unit test code.